examples/graphiql-parcel/src/index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+  <head>
+    <style>
+      body {
+        padding: 0;
+        margin: 0;
+        min-height: 100vh;
+      }
+      #root {
+        height: 100vh;
+      }
+    </style>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Parcel React Example</title>
+
+    <link rel="stylesheet" href="https://unpkg.com/graphiql/graphiql.min.css" />
```

Comment:
I happened to use the same method to import the css file since I couldn't import it from the module itself. `Post CSS Module is not an object  . . .` or something like that was the error. 

Do you happen to know why are we getting this error? 

P.s. it imports fine with `webpack` and `create-react-app`

Reply:
@daemon1024 can you open a bug for this error? i haven't seen it yet!